build.gradle.kts

```diff
@@ -42,7 +42,7 @@ tasks {
 
     patchPluginXml {
         sinceBuild.set("221")
-        untilBuild.set("222.*")
+        untilBuild.set("223.*")
```

Comment:
I should really get rid of the until build altogether to prevent needing to constantly update... but this is fine for this PR 👍